### ISSUE TITLE
[Diskuze] Zkusenosti - aktualni stav?

### ISSUE THREAD

**Issue by breznak** [l-ra/openeet-android#6]
Chtel bych se zeptat, testuje/zvazuje nekdo tuto aplikaci pro realne nasazeni ve svem podnikani? 
Od 1.12. bude EET platit pro prvni varku nestastniku.. 
(Pro vyvojare) V jakem je to stavu, prosim? 
Diky

**Comment by l-ra** [comment]
Tak, jak je aplikace koncipovana, neni urcena pro intenzivni pouzivani v provozu, kde se casto plati. Cilim ji spise na drobne zivnostniky, kteri udelaji par transakci za den. Pro me je cilem dosahnout zhruba obdobne funkcionality, jakou dnes ma aplikace EET+ (https://eetplus.cz), která je zdarma, ale není otevřená. Cilovym datem pro tuto funkcionalitu je spusteni druhe vlny v breznu pristiho roku.

Co se stavu tyka, aplikace eviduje do produkce, pamatuje si uctenky, ale do plne konformity s pozadavky ji chybi predani nejaka moznost predani uctenky (tisk, uloceni nekam do site). 

Ted par tydnu budu resit jeste jiny projekt souvisejici s EET, pak se k aplikaci vratim.
